tests/requirements.txt

```diff
@@ -1,2 +1,2 @@
-Django==4.2
-huey==2.4.5
+Django>=5.0a1,<5.1
+huey==2.5.0
```

Comment:
also upgrading Huey, it has some small API changes: https://github.com/coleifer/huey/blob/master/CHANGELOG.md